### DISCUSSION TITLE
Discussion Topics for Sep 13, 2021 Engineering Sync

### DISCUSSION THREAD

**Discussion by carlosfelix2** [CDCgov/prime-reportstream#2169] [Show and tell]
Discussion Topics for Sep 13, 2021 Engineering Sync

**Comment by carlosfelix2** [comment]
1. Proposal 13 - value sets and tables simplification @carlosfelix2 
2. Initial PR review process making its rounds for review - @carlosfelix2 
3. Quick tests in Kotlin - @carlosfelix2

**Comment by whytheplatypus** [comment]
Current git policy / practice. Where we are now, where we want to be, what we want to get out of it.
